mirrord/kube/src/api/container.rs

```diff
@@ -35,6 +35,7 @@ pub struct ContainerParams {
     pub name: String,
     pub gid: u16,
     pub port: u16,
+    pub extra_env: Vec<(String, String)>,
```

Comment:
:memo: 

Why is this here, how is this used? Docs please.

Reply:
You mean this `pub const AGENT_TLS_ENV: &str = "MIRRORD_AGENT_USE_TLS";` right?

Reply:
Yeah, previously this env was meant to be passed in the `extra_env` field. Now it's handled in this crate, based on `use_tls`